lib/ravelin/ato_login.rb

```diff
@@ -0,0 +1,13 @@
+module Ravelin
+  class AtoLogin < RavelinObject
+    attr_accessor :login,
+      :device,
+      :location
```

Comment:
`:login` ought to be `attr_reader` as you provide the write accessor explicitly below.

Reply:
As discussed in person, this breaks some of the reflection/serialization, so it was easier to follow the convention than refactor much of the gem :man_shrugging: